crates/swc_bundler/src/bundler/mod.rs

```diff
@@ -131,7 +131,9 @@ where
             }
         })
     }
-
+    pub fn is_external(&self, src: &JsWord) -> bool {
```

Comment:
Can you reduce the visibility of this method to `pub(crate)`?

Reply:
done